app/footer.tsx

```diff
@@ -1,3 +1,200 @@
+import Image from 'next/image';
+import { CiGlobe } from 'react-icons/ci';
+import { FaFacebookSquare, FaLinkedinIn, FaTwitter } from 'react-icons/fa';
+import { MdMailOutline, MdOutlineCall } from 'react-icons/md';
+import { SlLocationPin } from 'react-icons/sl';
+
+const quickLinks = [
```

Comment:
These objects will be defined inside the Footer function, not globally.

Also, compress all objects into one, like:
```
const items = {
    quickLinks = [ { text: "", link: "" ],
    about = [ { text: "", link: "" ],
    departments = [ { text: "", link: "" ],
}